packages/scalars/src/scalars/index.ts

```diff
@@ -1,4 +1,8 @@
 // namespace imports -- DO NOT REMOVE OR EDIT THIS COMMENT
+import * as Amount from "./Amount.js";
```

Comment:
Also I think that we need AmountPercentage Scalar

Reply:
it's already there: https://github.com/powerhouse-inc/powerhouse/blob/dspot-scalars/packages/scalars/src/scalars/AmountPercentage.ts